### DISCUSSION TITLE
Chromium Renderer process just crashed.

### DISCUSSION THREAD

**Discussion by jonathan-4iTester** [filiphric/cypress-plugin-api#97] [General]
Hey guys,

Following error when localhost running in chrome version: '108.0.5359.98' some test cases in my project

![image](https://user-images.githubusercontent.com/66831974/209331664-faef25ad-9ca5-4171-bbc5-76c9407c910b.png)

When my project is using "cypress-plugin-api": "^2.8.0" lib, the above error occurs. But when I use bahmutov's cy-api lib, these tests pass.

Could someone please help me?

**Comment by filiphric** [comment]
would it be possible to provide more details? what happens when you run this test in open mode?

**Reply by jonathan-4iTester** [reply]
This error occurs in scenarios other than a scenario outline, as shown in the image below:

![image](https://user-images.githubusercontent.com/66831974/209338671-9b27363d-49fa-45da-b8e2-e6a424b55184.png)

I suspect it's browser memory overflow. In this scenario outline, I'm using a cy.readFile() to read a json file for the request body.

Cypress version is 10.11.0

**Reply by jonathan-4iTester** [reply]
@filiphric, any news?

**Reply by filiphric** [reply]
Hey @jonathan-4iTester no, I haven't been working on the plugin these days but I will get back on it in next couple of weeks.

However, without reproducible example therw's not much I can do, as I cannot determine the cause of the problem you are facing. I'd be happy to help, but I need some hands-on information.
